k8s/example/values.yaml

```diff
@@ -73,52 +73,56 @@ global:
     # PCS CF1/CF3 Cluster, whereas some can simply ignore the usage.
     resources:
       limits:
-        cpu: 100m
-        memory: 128Mi
+        cpu: 200m
+        memory: 512Mi
       requests:
         cpu: 100m
         memory: 128Mi
-    svcPortNum: 18080
-    svcHealthPortNum: 18081
+    svcPortNum: 8080
```

Comment:
k8s/example should only be used for agent chart.

Reply:
All changes reverted for example/values.yaml as it is used only for agent test